shell/list/node.vue

```diff
@@ -209,23 +209,31 @@ export default {
                 class="mt-5"
               > {{ t('node.list.nodeLabels') }}:
                 <span
-                  v-if="isLabelsVisible || displayLabels(row).length < 8"
                   class="mt-5 labels"
                 >
                   <Tag
-                    v-for="(label, i) in displayLabels(row)"
+                    v-for="(label, i) in displayLabels(row).slice(0, 7)"
```

Comment:
Feels like a cleaner approach is to have a flag indicating if we are showing all or not.

The displayLabels method would then slice if necessary.

The button would just toggle the flag